.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.
```

Comment:
> Note that if you are not already a member of the bioconda project (meaning that you can't add this label)

I'd reverse this to: `Note that if you cannot add this label (meaning that you are not already a member of the bioconda project)`

> (please note if you'd like to be added to the bioconda project)

This aside is unfinished (may want to move it to its own sentence).

Reply:
Please feel free to change the wording :)